src/site/content/en/learn/css/selectors/index.md

```diff
@@ -0,0 +1,486 @@
+they act as if they are inserting a new element with CSS.
+Pseudo-elements are also syntactically different from pseudo-classes,
+because instead of using a single colon (`:`), we use a double colon (`::`).
+
+{% Aside %}
+A double colon (`::`) is what distinguishes a pseudo-element from a pseudo-class,
+but because this distinction wasn't present in older versions of CSS specs,
+browsers support a single colon for the original pseudo-elements,
+such as `:before` and `:after` to help with backwards compatibility with older browsers, like IE8.
+{% endAside %}
+
+```css
+.my-element::before {
+	content: "Prefix - ";
+}
+```
+
+As in the above demo where you prefixed a link's label with the type of file it was,
+you can use the `::before` pseudo-element to insert content **at the start of an element**,
+or the `::after` pseudo-element to insert content at the **end of an element**.
+
+Pseudo-elements aren't limited to inserting content, though.
+You can also use them to target specific parts of an element.
+For example, suppose you have a list.
+Use `::marker` to style each bullet point (or number) in the list
+
+```css
+/* Your list will now either have red dots, or red numbers */
+li::marker {
+	color: red;
+}
+```
+
+You can also use `::selection` to style the content that has been highlighted by a user.
+
+```css
+::selection {
+	background: black;
+	color: white;
+}
+```
+
+You can find out more about pseudo-elements in a later lesson.
+
+## Complex selectors
+
+You have already seen a vast array of selectors,
+but sometimes, you will need more *fine-grained control* with your CSS.
+This is where complex selectors step in to help.
+
+It's worth remembering at this point that although the following selectors give us more power,
+we can only  **cascade downwards**, selecting child elements.
+We are not able to target upwards and select a parent element.
+We cover what the cascade is and how it works in a later lesson.
+
+### Combinators
+
+A combinator is what sits between two selectors.
+For example, if the selector was `p > strong`, the combinator is the `>` character.
+The selectors which use these combinators help you select items based on their position in the document.
+
+#### Descendant combinator
+
+To understand descendant combinators,
+you need to first understand parent and child elements.
+
+```html
+<p>
+  A paragraph  of text with some <strong>bold text for emphasis</strong>.
+</p>
+```
+
+The parent element is the `<p>` which contains text.
+Inside that `<p>` element is a `<strong>` element, making its content bold.
+Because it is inside the `<p>`, it is a child element.
+
+A descendant combinator allows us to target a child element.
+This uses a space (` `) to instruct the browser to look for child elements:
+
+```css
+p strong {
+	color: blue
+}
+```
+
+This snippet selects all `<strong>` elements
+that are child elements of `<p>` elements only and makes them blue.
+
```

Comment:
maybe make a note here reminding about how `p strong {}` is a combinator, and if you wanted to instead make the color blue for all `p` and all `strong` to remember to add the comma for a comma separated list, i.e. `p, strong {}`

The space vs. comma separated might trip people up